src/main/scala/no/ndla/conceptapi/controller/ConceptController.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Part of NDLA concept_api.
+ * Copyright (C) 2019 NDLA
+ *
+ * See LICENSE
+ */
+
+
+package no.ndla.conceptapi.controller
+
+import org.scalatra.Ok
+
+trait ConceptController
+{
+
+  class ConceptController extends NdlaController
+  {
+    get("/")
+    {
+      Ok("Hellow World")
```

Comment:
```suggestion
      Ok("Hello World")
```
Berre for å være mest mulig pirkete...